libs/utils/trace.py

```diff
@@ -128,14 +128,8 @@ def __init__(self, platform, data_dir, events,
                           trace_format)
         self.__computeTimeSpan()
 
-        # Minimum and Maximum x_time to use for all plots
-        self.x_min = 0
-        self.x_max = self.time_range
```

Comment:
Actually, should not be better to initialize them with self.trace.window[1 and 2]?